examples/gallery/simple_differential_rotation.py

```diff
@@ -0,0 +1,70 @@
+"""
+============================
+Simple Differential Rotation
+============================
+
+The Sun is known to rotate differentially, meaning that the rotation rate
+near the poles (rotation period of approximately 35 days) is not the same as
+the rotation rate near the equator (rotation period of approximately 25 days).
+This is possible because the Sun is not a solid body. Though it is still poorly
+understood, it is fairly well measured and must be taken into account
+when comparing observations of features on the Sun over time.
+A good review can be found in Beck 1999 Solar Physics 191, 47–70.
+This example illustrates solar differential rotation.
+"""
+
+##############################################################################
+# Start by importing the necessary modules.
+from datetime import timedelta
```

Comment:
we should import `division` from `__future__` here for consistency between python 2 and 3. I would also argue we should import `print_function` for completeness as well. (in case someone downloads the file and runs it).